### ISSUE TITLE
v-model is setting value to `null`

### ISSUE THREAD

**Issue by SPQRInc** [dm4t2/vue-currency-input#12]
Hello everybody,

I got this object, which is loaded via API before the page is rendered:

```
{ "id": 1, "title": "Test", "price": 9, "setup": 0}
```

Now if I add this to my Vue project:

```
<currency-input id="product-price"
                                            name="product-price"
                                            v-validate="'required'"
                                            :class="{'form-control': true, 'is-invalid': errors.has('product-price')}"
                                            :currency="currency"
                                            v-model="product.price"/>
```

I am getting no value - `product.price` is empty now:

 ```
{ "id": 1, "title": "Test", "price": null, "setup": 0}
```

If I remove the `currency-input`, the object is correct again:

 ```
{ "id": 1, "title": "Test", "price": 9, "setup": 0}
```

**Comment by SPQRInc** [comment]
Additional info: It works fine if I use `vue-currency-input: "1.3.1"`

**Comment by dm4t2** [comment]
Can you provide a [CodeSandbox](https://codesandbox.io/) to reproduce this issue?

**Comment by SPQRInc** [comment]
Sure (well - I am using Codepen, but I hope this is working for you, too)

This is the pen: https://codepen.io/spqrinc/pen/gJexvR/

In the JS-field, you may find this code:

```
const StartPage = {
	template: `
    <div>
		<h2>Raw data</h2>
		<div>
			{{product}}
		</div>
		<hr />
		<h2>Vue-Currency-Input</h2>
		<currency-input v-model="product.price"/> <!--Remove this to see the result of product above-->
	</div>
  `,
```

If you remove `<currency-input v-model="product.price"/>` here, you can see, that the raw data returns `{ "price": 1.09 }`. If currency-input is inserted, the raw data is `{ "price": null }`.

**Comment by dm4t2** [comment]
Thank you, I will fix this issue in the next release.

**Comment by SPQRInc** [comment]
Thanks a lot for all the work :-)

Unfortunately I can not test it at the moment (https://github.com/dm4t2/vue-currency-input/issues/13) .

If you find some time, I would be very happy if you could check out the linked issue so that I can test the fix in 1.5.0 :-)

**Comment by SPQRInc** [comment]
In `1.5.1` it works as expected. Thanks!
